focus-ios/README.md

```diff
@@ -8,7 +8,7 @@ Download on the [App Store](https://itunes.apple.com/app/id1055677337).
 
 ## Main Branch
 
-This branch works with Xcode 15.1 and supports iOS 13.0 and newer.
+This branch works with Xcode 15.3 and supports iOS 15.0 and newer.
```

Comment:
@OrlaM Could you please confirm if we are supporting iOS 15.x, 16.x and 17.x? I think 13.x is no longer supported.